bigbluebutton-html5/imports/ui/components/audio/audio-graphql/hooks/useToggleVoice.ts

```diff
@@ -14,9 +14,10 @@ const useToggleVoice = () => {
 
   const toggleVoice = async (userId?: string | null, muted?: boolean | null) => {
     let shouldMute = muted;
-    const userToMute = userId ?? Auth.userID;
+    let userToMute = userId ?? Auth.userID;
 
     if (muted === undefined || muted === null) {
```

Comment:
It's confusing to assume it's the current user when `muted` is null.

My suggestion is to set both parameters as required!

And if you want you can create a different function `toggleMutedForCurrentUser` that will call this function passing `userId` and `muted` of the current user.
It will make things really clearer for future maintenance.